src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -573,7 +573,7 @@ public Boolean call() throws Exception
 
             // failed to get connection from db, sleep and retry
             quietlySleep(sleepBackoff);
-            sleepBackoff = Math.min(connectionTimeout / 2, (long) (sleepBackoff * 1.3));
+            sleepBackoff = Math.min(connectionTimeout / 5, (long) (sleepBackoff * 1.3));
```

Comment:
agreed. lets see another angle.
for 1000s. till 5th failure 200*1.3 plays good role. by then, it already slept for (sum) 1009ms
by then the client already notified of 'timeout failure'
considering both setup (high / low value of connection timeout), probably reasonable value to sleep minimum 250 mills - max 5 seconds?